docs/arguments.md

```diff
@@ -61,8 +61,6 @@ The ingester query API was improved over time, but defaults to the old behaviour
 
    Time since the last sample after which a time series is considered stale and ignored by expression evaluations.
 
-## Query Frontend
-
 - `-querier.align-querier-with-step`
```

Comment:
All the flags below are also part of the frontend and not the querier. They should be updated in `pkg/querier/queryrange/roundtrip.go`